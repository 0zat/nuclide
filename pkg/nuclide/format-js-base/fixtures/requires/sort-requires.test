require('ccc-side-effect');
var AAA = require('AAA');
var bbb = require('bbb');


require('bbb-side-effect');
var CCC = require('CCC');
var aaa = require('aaa');


require('aaa-side-effect');
var BBB = require('BBB');
var ccc = require('ccc');

// Make sure they are used.
aaa(AAA);
bbb(BBB);
ccc(CCC);
