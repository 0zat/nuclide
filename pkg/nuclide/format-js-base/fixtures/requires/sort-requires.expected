require('aaa-side-effect');
require('bbb-side-effect');
require('ccc-side-effect');

var AAA = require('AAA');
var BBB = require('BBB');
var CCC = require('CCC');

var aaa = require('aaa');
var bbb = require('bbb');
var ccc = require('ccc');

// Make sure they are used.
aaa(AAA);
bbb(BBB);
ccc(CCC);
